### ISSUE TITLE
chore(tabs): classMap in Tabset `has changed after it was checked`

### ISSUE THREAD

**Issue by NN77** [valor-software/ngx-bootstrap#70]
> EXCEPTION: Expression 'classMap in Tabset@1:20' has changed after it was checked. Previous value: '[object Object]'. Current value: '[object Object]' in [classMap in Tabset@1:20]

``` html
<tabset>
  <tab heading="Tab 1">Tab 1 content</tab>
  <tab>
    <template tab-heading="">Tab 2</template>Tab 2 content
  </tab>
</tabset>
```

``` javascript
...
import {Alert, Collapse, TAB_COMPONENTS, DROPDOWN_DIRECTIVES} from 'ng2-bootstrap/ng2-bootstrap';

@Component({
    ...
    directives: [CORE_DIRECTIVES, NgFor, Alert, Collapse, TAB_COMPONENTS, DROPDOWN_DIRECTIVES]
})
export class header {
    public isCollapsed:boolean = true;
    private disabled:boolean = false;
    private status:{isopen:boolean} = {isopen: false};

    private toggleDropdown($event:MouseEvent):void {
        $event.preventDefault();
        $event.stopPropagation();
        this.status.isopen = !this.status.isopen;
    }

    private tabs:Array<any> = [
        {title: 'Dynamic Title 1', content: 'Dynamic content 1'},
        {title: 'Dynamic Title 2', content: 'Dynamic content 2', disabled: true}
    ];

    constructor() {

    }
}
```

Following updated examples with TAB_COMPONENTS insted of TAB_DIRECTIVES only above issue remained using ng2 beta

**Comment by valorkin** [comment]
Thanks, for issue, I have seen such issues before

**Comment by valorkin** [comment]
seems to be same as this
https://github.com/valor-software/ng2-bootstrap/issues/52

**Comment by valorkin** [comment]
fixed in v1.0.8 with beta.12
